### PR TITLE
fix(logs): insert message without an ID

### PR DESCRIPTION
**Pull request by bodinsamuel**
## Describe your changes

Fixes https://linear.app/nango/issue/NAN-1244/insert-messages-without-an-id

Now that we have Elasticsearch in prod I can focus a bit more time on performance quick wins.

- Insert messages (not operation) without an ID
I knew this from the beginning but I just wanted to simplify the code. They recommend not using an ID because if you pass one, all the instances need to search in their whole catalog to see if this ID already exists. Whereas if you don't send it, they will autogenerate an ordered id without checking. This should help but no clue how much.

- Clarify types between OperationRow and MessageRow which were different already but a bit mixed up
It's not perfect yet since most fields are optional 
- Fix some type errors

### CODE REVIEW

**Review by TBonnin — APPROVED** [review]
curious to see how much it helps

File: packages/logs/lib/models/helpers.ts [hunk 1]
```diff
@@ -50,8 +42,45 @@ export function getFormattedMessage(
         syncConfigName: syncConfig?.name || data.syncConfigName || null,
 
         jobId: data.jobId || null,
+        meta: meta || data.meta || null,
 
         userId: user?.id || data.userId || null,
+        parentId: null
+    };
+}
+export function getFormattedMessage(data: Partial<MessageRow>, { meta }: FormatMessageData = {}): MessageRow {
```
Comment (TBonnin) [inline packages/logs/lib/models/helpers.ts:51]: should this method return a `Pick<MessageRow, ...>` to make it clear it is only setting a subset of the fields?

Comment (bodinsamuel) [reply packages/logs/lib/models/helpers.ts:51]: it's still returning everything, just more null than before :D

Comment (TBonnin) [reply packages/logs/lib/models/helpers.ts:51]: forget what I said. I thought the function was only used in getFormattedOperation but it is called from a lot of places

**Pull request merged**